optaplanner-integration/optaplanner-workbench-models/optaplanner-workbench-models-core/pom.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.optaplanner</groupId>
+    <artifactId>optaplanner-workbench-models</artifactId>
+    <version>8.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>optaplanner-workbench-models-core</artifactId>
+  <packaging>jar</packaging>
+
+  <name>OptaPlanner core GWT wrapper for OptaPlanner Workbench</name>
+  <description>
+    OptaPlanner solves planning problems.
+    This lightweight, embeddable planning engine implements powerful and scalable algorithms
+    to optimize business resource scheduling and planning.
+
+    This module wraps OptaPlanner core classes used by OptaPlanner Workbench.
+  </description>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.jboss.errai.bom</groupId>
+        <artifactId>errai-internal-bom</artifactId>
+        <version>${version.org.jboss.errai}</version>
```

Comment:
Don't we get this version from the kie-parent pom already?

Reply:
These deps are not declared in kie-parent, see https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/pom.xml#L1923. Not sure whether we follow any policy in adding the deps there, or we just add those on demand?

Reply:
@ge0ffrey Discussed this with Peter S and current solution looks good. The errai dependencies were included in kie-parent just for sake of excluding juice. Adding errai-bom to kie-parent would indirectly introduce optaplanner-core -> errai dependency (as maven needs to resolve errai artifact), that's probably not what we want. This way the errai dependency is limited only to module which needs it (optaplanner-workbench-models-core).